dev-libs/opensp/patches/opensp-1.5.2.patchset

```diff
@@ -0,0 +1,2457 @@
++	GUESS=sparc-unknown-lynxos$UNAME_RELEASE
++	;;
+     rs6000:LynxOS:2.*:*)
+-	echo rs6000-unknown-lynxos${UNAME_RELEASE}
+-	exit ;;
+-    PowerPC:LynxOS:2.*:* | PowerPC:LynxOS:3.[01]*:* | PowerPC:LynxOS:4.0*:*)
+-	echo powerpc-unknown-lynxos${UNAME_RELEASE}
+-	exit ;;
++	GUESS=rs6000-unknown-lynxos$UNAME_RELEASE
++	;;
++    PowerPC:LynxOS:2.*:* | PowerPC:LynxOS:3.[01]*:* | PowerPC:LynxOS:4.[02]*:*)
++	GUESS=powerpc-unknown-lynxos$UNAME_RELEASE
++	;;
+     SM[BE]S:UNIX_SV:*:*)
+-	echo mips-dde-sysv${UNAME_RELEASE}
+-	exit ;;
++	GUESS=mips-dde-sysv$UNAME_RELEASE
++	;;
+     RM*:ReliantUNIX-*:*:*)
+-	echo mips-sni-sysv4
+-	exit ;;
++	GUESS=mips-sni-sysv4
++	;;
+     RM*:SINIX-*:*:*)
+-	echo mips-sni-sysv4
+-	exit ;;
++	GUESS=mips-sni-sysv4
++	;;
+     *:SINIX-*:*:*)
+ 	if uname -p 2>/dev/null >/dev/null ; then
+ 		UNAME_MACHINE=`(uname -p) 2>/dev/null`
+-		echo ${UNAME_MACHINE}-sni-sysv4
++		GUESS=$UNAME_MACHINE-sni-sysv4
+ 	else
+-		echo ns32k-sni-sysv
++		GUESS=ns32k-sni-sysv
+ 	fi
+-	exit ;;
+-    PENTIUM:*:4.0*:*) # Unisys `ClearPath HMP IX 4000' SVR4/MP effort
+-                      # says <Richard.M.Bartel@ccMail.Census.GOV>
+-        echo i586-unisys-sysv4
+-        exit ;;
++	;;
++    PENTIUM:*:4.0*:*)	# Unisys `ClearPath HMP IX 4000' SVR4/MP effort
++			# says <Richard.M.Bartel@ccMail.Census.GOV>
++	GUESS=i586-unisys-sysv4
++	;;
+     *:UNIX_System_V:4*:FTX*)
+ 	# From Gerald Hewes <hewes@openmarket.com>.
+ 	# How about differentiating between stratus architectures? -djm
+-	echo hppa1.1-stratus-sysv4
+-	exit ;;
++	GUESS=hppa1.1-stratus-sysv4
++	;;
+     *:*:*:FTX*)
+ 	# From seanf@swdc.stratus.com.
+-	echo i860-stratus-sysv4
+-	exit ;;
++	GUESS=i860-stratus-sysv4
++	;;
+     i*86:VOS:*:*)
+ 	# From Paul.Green@stratus.com.
+-	echo ${UNAME_MACHINE}-stratus-vos
+-	exit ;;
++	GUESS=$UNAME_MACHINE-stratus-vos
++	;;
+     *:VOS:*:*)
+ 	# From Paul.Green@stratus.com.
+-	echo hppa1.1-stratus-vos
+-	exit ;;
++	GUESS=hppa1.1-stratus-vos
++	;;
+     mc68*:A/UX:*:*)
+-	echo m68k-apple-aux${UNAME_RELEASE}
+-	exit ;;
++	GUESS=m68k-apple-aux$UNAME_RELEASE
++	;;
+     news*:NEWS-OS:6*:*)
+-	echo mips-sony-newsos6
+-	exit ;;
++	GUESS=mips-sony-newsos6
++	;;
+     R[34]000:*System_V*:*:* | R4000:UNIX_SYSV:*:* | R*000:UNIX_SV:*:*)
+-	if [ -d /usr/nec ]; then
+-	        echo mips-nec-sysv${UNAME_RELEASE}
++	if test -d /usr/nec; then
++		GUESS=mips-nec-sysv$UNAME_RELEASE
+ 	else
+-	        echo mips-unknown-sysv${UNAME_RELEASE}
++		GUESS=mips-unknown-sysv$UNAME_RELEASE
+ 	fi
+-        exit ;;
++	;;
+     BeBox:BeOS:*:*)	# BeOS running on hardware made by Be, PPC only.
+-	echo powerpc-be-beos
+-	exit ;;
++	GUESS=powerpc-be-beos
++	;;
+     BeMac:BeOS:*:*)	# BeOS running on Mac or Mac clone, PPC only.
+-	echo powerpc-apple-beos
+-	exit ;;
++	GUESS=powerpc-apple-beos
++	;;
+     BePC:BeOS:*:*)	# BeOS running on Intel PC compatible.
+-	echo i586-pc-beos
+-	exit ;;
++	GUESS=i586-pc-beos
++	;;
++    BePC:Haiku:*:*)	# Haiku running on Intel PC compatible.
```

Comment:
config.guess should only be patched with the Haiku cases.

Reply:
Thanks, fixed